resources/views/edit_car.blade.php

```diff
@@ -47,7 +47,7 @@
               <select name="category_id" class="form-control">
 
                 @foreach ($categories as $item)
-                    <option value="{{$item->id}}" {{($car->category_id== $item->id)?"selected": "" }}>{{$item->category}}</option>
+                    <option value="{{$item->id}}"  @selected($car->category_id== $item->id)>{{$item->category}}</option>
```

Comment:
Oooh i have forget this part,,

try this one 
```php
@selected(old('category_id', $car->category_id) == $category->id)
```

and in add car
```php
@selected(old('category_id') == $category->id)
```

do you notice the difference?

Reply:
wow , i got it . thank you.